static/app/components/replays/useReplaysCount.spec.tsx

```diff
@@ -131,6 +111,65 @@ describe('useReplaysCount', () => {
     });
   });
 
+  it('should request the count for a group only once', async () => {
+    const replayCountRequest = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/issue-replay-count/`,
+      method: 'GET',
+      body: {},
+    });
+
+    const {result, rerender, waitForNextUpdate} = reactHooks.renderHook(useReplaysCount, {
+      initialProps: {
+        organization,
+        projectIds,
+        groupIds: mockGroupIds,
+      },
+    });
+
+    await waitForNextUpdate();
+
+    expect(replayCountRequest).toHaveBeenCalledTimes(1);
+    expect(replayCountRequest).toHaveBeenCalledWith(
+      '/organizations/org-slug/issue-replay-count/',
+      expect.objectContaining({
+        query: {
+          query: `issue.id:[123,456]`,
+          statsPeriod: '14d',
+          project: [2],
+        },
+      })
+    );
+    expect(result.current).toEqual({
+      123: 0,
+      456: 0,
+    });
+
+    rerender({
+      organization,
+      projectIds,
+      groupIds: [...mockGroupIds, '789'],
+    });
```

Comment:
if we re-render with same groupIds (e.g. mockGroupIds), will `replayCountRequest` get called an additional time?

Reply:
it will not! 

there's this check to prevent it:
```
      if (!groupsToFetch.length) {
        return null;
      }
``` 

but i notice now there's no test for that. if we had some kind of thing showing if a line-is-executed by the test that'd be cool. Like a coverage of code thing it'd be harder to miss things!